README.md

```diff
@@ -1502,6 +1502,7 @@ Table of Contents
   * [thedev.id](https://thedev.id) — A free thedev.id subdomain for developers.
   * [Themeselection](https://themeselection.com/) — Selected high quality, modern design, professional and easy-to-use Free Admin Dashboard Template,
 HTML Themes and UI Kits to create your applications faster!
+  * [Voucherify](https://voucherify.io) — Voucherify is an API-based promotion and loyalty management software that empowers marketers to quickly launch and efficiently manage personalized coupons, gift cards, in-cart promotions, giveaways, referral, and loyalty programs.
```

Comment:
While yes, this has a free tier, I dont think it's really suitable for the scope of the list.